frontend/tests/integration/specs/work-packages/details-pane/details-pane-editable-spec.js

```diff
@@ -26,43 +26,18 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-var chai = require('chai');
-var chaiAsPromised = require('chai-as-promised');
-
-chai.use(chaiAsPromised);
-var expect = chai.expect;
-
-
-var WorkPackageDetailsPane = require('./pages/work-package-details-pane.js');
+var expect = require('../../../spec_helper.js').expect, 
+    detailsPaneHelper = require('./details-pane-helper.js');
 
 /*jshint expr: true*/
-
-describe('OpenProject', function() {
-  function loadPane(workPackageId) {
-    var page = new WorkPackageDetailsPane(workPackageId, 'overview');
-    page.get();
-    browser.waitForAngular();
-  }
-
-  describe('pane itself', function() {
-    beforeEach(function() {
-      loadPane(819);
-    });
-
-    it('should be visible', function() {
-      expect($('.work-packages--details').isPresent()).to.eventually.be.true;
-    });
-  });
-
-
+describe('OpenProject', function(){
   describe('editable', function() {
-
     function behaveLikeEmbeddedDropdown(name, correctValue) {
       context('behaviour', function() {
         var editor = $('[ined-attribute=\'' + name + '\'] .inplace-editor');
 
         beforeEach(function() {
-          loadPane(819);
+          detailsPaneHelper.loadPane(819, 'overview');
```

Comment:
@hirtie-maxim @0xF013 can't we move the `browser.waitForAngular()` into the page object instead of adding an extra helper file here?